plugins/nf-ga4gh/src/main/nextflow/ga4gh/tes/executor/TesTaskHandler.groovy

```diff
@@ -190,6 +191,9 @@ class TesTaskHandler extends TaskHandler {
         body.addInputsItem(inItem(scriptFile))
         body.addInputsItem(inItem(wrapperFile))
 
+        Path remoteBinDir = executor.getRemoteBinDir()
+        body.addInputsItem(inItem(remoteBinDir, remoteBinDir.toString(), true))
+
```

Comment:
Updated here to only pass the bin directory as TES Inputs (passes the directory itself as opposed to individual files, need to confirm that this is compatible with the TES Schema).

Reply:
Confirmed with @kellrott that passing a directory as a TES input is compatible with the schema (as long as all items in that directory are recursively add as TES inputs by the executor).